lib/openshift/pod_disruption_budget.rb

```diff
@@ -0,0 +1,6 @@
+module BushSlicer
+  class PodDisruptionBudget < ClusterResource
```

Comment:
from output
```
namespace: openshift-kube-descheduler-operator
```

So this object should be `ProjectResource`, not `ClusterResource`.

Reply:
Thought so, but me missed to update as well, thanks !!